Gemfile

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+# gem "rails"
+
+gem "exif", "~> 2.2"
```

Comment:
If it's just a project like this, I would recommend not specifying a version in the Gemfile. We usually do this in a large app when we know we need to be careful about the version for a reason (the reasons vary but as an example, if the gem requires another version to be upgraded first and we can't upgrade it yet). In this case, I would just use:

```ruby
gem "exif"
gem "thor"
```

Reply:
Thanks, makes sense to me